Tests/MusicXMLTests/ScoreTests/Reve/ReveParsingExpectation.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ReveParsingExpectation.swift
+//  
+//
+//  Created by Ben Lu on 10/13/19.
+//
+
+import Foundation
```

Comment:
Can we do without the `Foundation` import?

Reply:
sure